packages/lesswrong/components/users/ReviewVotingPage.tsx

```diff
@@ -160,8 +160,8 @@ const ReviewVotingPage = ({classes}) => {
   });
 
   const [submitVote] = useMutation(gql`
-    mutation submitReviewVote($postId: String, $qualitativeScore: Int, $quadraticScore: Int, $comment: String) {
-      submitReviewVote(postId: $postId, qualitativeScore: $qualitativeScore, quadraticScore: $quadraticScore, comment: $comment) {
+    mutation submitReviewVote($postId: String, $qualitativeScore: Int, $quadraticChange: Int, $setQuadraticScore: Int, $comment: String) {
+      submitReviewVote(postId: $postId, qualitativeScore: $qualitativeScore, quadraticChange: $quadraticChange, comment: $comment, setQuadraticScore: $setQuadraticScore) {
```

Comment:
I'd find it less confusing if "setQuadraticScore" was called "newQuadraticScore". I really expect setQuadraticScore to be a function